### DISCUSSION TITLE
Run button

### DISCUSSION THREAD

**Discussion by jett-kim** [codesandbox/sandpack#871] [General]
Hi, I am trying to run code only by run button.
If i put `options={ autorun: false }` run button appears in first time.
But if i fire once it just disappears and works same as the autorun: true.
Can i make run button to exist permanently?

**Comment by danilowoz** [comment]
I believe you need a new prop, which I just introduced: 
```js
import { Sandpack } from "@codesandbox/sandpack-react";

export default function App() {
  return (
    <Sandpack 
      template="react"
      options={{
        autorun: false, // whether or not the bundling process should start automatically
        autoReload: false // whether the component should automatically reload when changes are made to the code.
      }}
    />
  );
}
```

https://github.com/codesandbox/sandpack/pull/877
